regionService/srv/www/regionService/regionInfo.py

```diff
@@ -134,12 +134,12 @@ def create_smt_region_map(conf):
         region_name_to_smt_data_map[section] = smtInfo
         for ipRange in region_public_ip_ranges.split(','):
```

Comment:
Todo: Unrelated but note to fix case.

```ip_range```, ```smt_info``` etc

Reply:
variable names get fixed when the IPv6 support gets implemented, also hopefully this week